prov/efa/test/efa_unit_test_cq.c

```diff
@@ -0,0 +1,249 @@
+#include "efa_unit_tests.h"
+
+/**
+ * @brief verify fi_cq_read() works with empty device CQ for given endpoint type
+ *
+ * When CQ is empty, fi_cq_read() should return -FI_EAGAIN.
+ *
+ * @param[in]		ep_type		endpoint type, can be FI_EP_DGRAM or FI_EP_RDM
+ */
+static
+void test_cq_read_empty_cq(enum fi_ep_type ep_type)
```

Comment:
Since this is not an actual test, suggest removing the `test_*` prefix.

Reply:
I think it is still a test but an implementation, so I changed it to `test_impl`.

Any suggestion are welcome!